src/IB/IBHydrodynamicSurfaceForceEvaluator.cpp

```diff
@@ -64,7 +64,38 @@ static const int GLEVELSETG = 1;
 static const int GVELOCITYG = 1;
 static const int GPRESSUREG = 1;
 static const int GVISCOSITYG = 1;
-} // namespace
+
+inline int
+sign(const double X)
+{
+    return ((X > 0) ? 1 : ((X < 0) ? -1 : 0));
+} // sign
+
+inline void
+get_physical_coordinate(IBTK::Vector3d& side_coord, Pointer<Patch<NDIM> > patch, const SideIndex<NDIM>& side_idx)
```

Comment:
@amneetb it looks like the same code is showing up in several different places. Should this go into, e.g., `IndexUtilities`?

Reply:
👍

Reply:
Actually that method is already in `IndexUtilities` but with a fixed return type. I will template on the return type.